app/wizards/claims/edit_request_clawback_wizard.rb

```diff
@@ -0,0 +1,45 @@
+module Claims
+  class EditRequestClawbackWizard < BaseWizard
+    attr_reader :claim, :mentor_training_id
+
+    def initialize(claim:, params:, state:, mentor_training_id:, current_step:)
+      @claim = claim
+      @mentor_training_id = mentor_training_id
+      super(state:, params:, current_step:)
+    end
+
+    def define_steps
+      add_step(RequestClawbackWizard::MentorTrainingClawbackStep,
+               { mentor_training_id: mentor_training.id },
+               :mentor_training_id)
+    end
+
+    def mentor_training
+      @mentor_training ||= claim.mentor_trainings.not_assured.find(mentor_training_id)
```

Comment:
Given you have `mentor_trainings` on line 21 you should reduce this to 
```suggestion
      @mentor_training ||= mentor_trainings.find(mentor_training_id)
```